proxy/contracts/mainnet/CommunityPool.sol

```diff
@@ -85,11 +85,16 @@ contract CommunityPool is Twin {
     )
         external
         onlyMessageProxy
-        returns (bool)
+        returns (uint)
     {
-        require(activeUsers[user][schainHash], "User should be active");
```

Comment:
Are you sure that you want to remove this check? It allows blocked users to send messages.